--- GraphicsExtensions.cs	2016-09-23 01:27:59.000000000 +0300
+++ GraphicsExtensions.cs.patched	2016-09-23 01:16:25.000000000 +0300
@@ -914,7 +914,7 @@
 #else
             var error = GL.GetError();
 #endif
-            if (error != ErrorCode.NoError)
+            if (error != ErrorCode.NoError && error != ErrorCode.InvalidFramebufferOperationExt)
                 throw new MonoGameGLException("GL.GetError() returned " + error.ToString());
         }
 #endif
