diff --git "a/C:\\Users\\SEVENO~1\\AppData\\Local\\Temp\\TortoiseGit\\Con94DF.tmp\\ContentManager-5912837-left.cs" "b/D:\\Jypeli\\MonoKaannos\\MonoGame\\MonoGame.Framework\\Content\\ContentManager.cs"
index b639300..ea6823b 100644
--- "a/C:\\Users\\SEVENO~1\\AppData\\Local\\Temp\\TortoiseGit\\Con94DF.tmp\\ContentManager-5912837-left.cs"
+++ "b/D:\\Jypeli\\MonoKaannos\\MonoGame\\MonoGame.Framework\\Content\\ContentManager.cs"
@@ -361,6 +361,11 @@ namespace Microsoft.Xna.Framework.Content
             {
                 return EffectReader.Normalize(assetName);
             }
+            else if ( ( typeof( T ) == typeof( string[] ) ) )
+            {
+                return ArrayReader<string>.Normalize( assetName, new string[] { ".txt" } );
+            }
+
             return null;
         }
 
@@ -404,6 +409,26 @@ namespace Microsoft.Xna.Framework.Content
                     return new Effect(this.graphicsDeviceService.GraphicsDevice, data);
                 }
             }
+
+            else if ( ( typeof( T ) == typeof( string[] ) ) )
+            {
+                List<string> list = new List<string>();
+
+                using ( Stream assetStream = TitleContainer.OpenStream( assetName ) )
+                {
+                    using ( StreamReader reader = new StreamReader( assetStream ) )
+                    {
+                        while ( !reader.EndOfStream )
+                        {
+                            string line = reader.ReadLine();
+                            list.Add( line );
+                        }
+                    }
+                }
+
+                return list.ToArray();
+            }
+
             return null;
         }
 
