diff --git "a/C:\\Users\\Tomi\\AppData\\Local\\Temp\\TortoiseGit\\Gra5EC7.tmp\\GraphicsExtensions-74fef2d-left.cs" "b/C:\\Versiohallinta\\MonoJypeli\\MonoGame\\MonoGame.Framework\\Graphics\\GraphicsExtensions.cs"
index a96fc03..be6a59c 100644
--- "a/C:\\Users\\Tomi\\AppData\\Local\\Temp\\TortoiseGit\\Gra5EC7.tmp\\GraphicsExtensions-74fef2d-left.cs"
+++ "b/C:\\Versiohallinta\\MonoJypeli\\MonoGame\\MonoGame.Framework\\Graphics\\GraphicsExtensions.cs"
@@ -879,7 +879,7 @@ namespace Microsoft.Xna.Framework.Graphics
 #else
             var error = GL.GetError();
 #endif
-            if (error != ErrorCode.NoError)
+            if (error != ErrorCode.NoError && error != ErrorCode.InvalidFramebufferOperationExt)
                 throw new MonoGameGLException("GL.GetError() returned " + error.ToString());
         }
 #endif
