diff --git a/MonoGame.Framework/Desktop/OpenTKGameWindow.cs b/MonoGame.Framework/Desktop/OpenTKGameWindow.cs
index d378722..f91754f 100644
--- a/MonoGame.Framework/Desktop/OpenTKGameWindow.cs
+++ b/MonoGame.Framework/Desktop/OpenTKGameWindow.cs
@@ -54,7 +54,7 @@ using OpenTK.Graphics;
 
 namespace Microsoft.Xna.Framework
 {
-    class OpenTKGameWindow : GameWindow, IDisposable
+    public class OpenTKGameWindow : GameWindow, IDisposable
     {
         private bool _isResizable;
         private bool _isBorderless;
